index.bs

```diff
@@ -723,7 +723,7 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
 
 1. Let |cache| be the user agent's [=attribution source cache=].
 1. [=list/Remove=] all entries in |cache| where all of the following are true:
-    * the entry's [=attribution source/attribution destination=] and |source|'s [=attribution source/attribution destination=] are equal.
+    * the entry's [=attribution source/attribution destination=] is [=same site=] with |source|'s [attribution source/attribution destination].
```

Comment:
From the same site link:

> Two origins, A and B, are said to be same site...

But `attribution destination` is a site, so I'm not sure the operation is defined here, and `equals` may actually be correct.

Reply:
Yeah, that's right. Currently there are two places we use `same site` for a site, will replace them to `equals` as well.